spotify_tensorflow/flags.py

```diff
@@ -30,20 +31,35 @@ class Flags(object):
     @staticmethod
     def register_dataset_flags():
         logging.info("Registering Dataset flags")
+
+        flags.DEFINE_string("train_subdir", "train",
+                            "Location of training TFRecords, with the training set dir.")
+
+        flags.DEFINE_string("eval_subdir", "eval",
+                            "Location of eval TFRecords, with the training set dir.")
+
+        # Dataset API parameters
```

Comment:
I wonder whether we should have a namespace here? Like `datasets_ batch_size` and `datasets_ prefetch_buffer_size`? What do you think?

Reply:
I don't think it's super necessary, for example the meaning of the `batch_size` parameter totally align with what ML ppl think this parameter means.